packages/integration-test-app/windows/integrationtest.sln

```diff
@@ -54,12 +54,12 @@ Global
 		..\..\..\vnext\Shared\Shared.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|ARM64 = Debug|ARM64
```

Comment:
Intentional change? Note: Looks totally fine, just asking in case this finally affects the default config VS opens in :)

Reply:
Very intentional.

Visual Studio **insists** in ordering by platform alphabetically, even though probably no one uses ARM64 as their main/default target.

This only happens whenever you add or remove an item from the solution within the IDE.